corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -29,7 +29,7 @@ def trans(name, langs=None, include_lang=True, use_delim=True, prefix=False, esc
     else:
         tag = lambda lang: ""
     for lang in langs:
-        if lang in name and name[lang]:
+        if lang in name and name[lang] is not None:
```

Comment:
Will add the comment.
ya.. it can't be changed to older simpler statement unless there is another way to address the issue i highlighted.